packages/block-editor/src/store/actions.js

```diff
@@ -658,6 +658,302 @@ export const synchronizeTemplate = () => ( { select, dispatch } ) => {
 	dispatch.resetBlocks( updatedBlockList );
 };
 
+function mapRichTextSettings( attributeDefinition ) {
+	const {
+		multiline: multilineTag,
+		__unstableMultilineWrapperTags: multilineWrapperTags,
+		__unstablePreserveWhiteSpace: preserveWhiteSpace,
+	} = attributeDefinition;
+	return {
+		multilineTag,
+		multilineWrapperTags,
+		preserveWhiteSpace,
+	};
+}
+
+/**
+ * Delete the current selection.
+ *
+ * @param {boolean} isForward
+ */
+export const deleteSelection = ( isForward ) => ( {
+	registry,
+	select,
+	dispatch,
+} ) => {
+	const selectionAnchor = select.getSelectionStart();
+	const selectionFocus = select.getSelectionEnd();
+
+	if ( selectionAnchor.clientId === selectionFocus.clientId ) return;
```

Comment:
should we not wrap these in `{}`s to avoid making it easy to recreate some infamous bugs due to the missing brackets?

```ts
// oops!
if ( … )
	someNewThing();
	return;
```

Reply:
Prettier will decrease indentation, so imho it's clear when you add lines. If we don't like this style, we should probably add the relevant rule to the linter.

Reply:
just raising it because I thought the practice was long abandoned because of its risk. I hadn't seen it in Gutenberg before and it seems riskier still to rely on the auto-formatter to avoid code patterns that invite bugs. carry on 🙇‍♂️

(by the way, it would seem fine to add this to the linter if other people are also leery of single-line `if` blocks)

Reply:
Right, I'm fine with avoiding it. Maybe we should add it to the linter though, so it's "official".